resources/qml/device-verification/Success.qml

```diff
@@ -15,7 +15,7 @@ Pane {
             Layout.fillHeight: true
             Layout.fillWidth: true
             wrapMode: Text.Wrap
-            text: qsTr("Verification successful! Both sides verified their devices!")
+            text: qsTr("Verification successful! Both sides verified their device!")
```

Comment:
If this is a user verification, this is not correct. You may have verified multiple devices now. It may make sense to explicitly list, what was verified right now, but I don't think you can do that with just a translation change. It needs conditions on the different cases and maybe some backend changes.

Reply:
All right, then :)